packages/core/http/core-http-router-server-internal/src/request.ts

```diff
@@ -59,15 +59,18 @@ export class CoreKibanaRequest<
    */
   public static from<P, Q, B>(
     req: RawRequest,
-    routeSchemas: RouteValidator<P, Q, B> | RouteValidatorFullConfig<P, Q, B> = {},
+    routeSchemas:
+      | RouteValidator<P, Q, B>
+      | RouteValidatorFullConfig<P, Q, B>
+      | undefined = undefined,
     withoutSecretHeaders: boolean = true
   ) {
-    const routeValidator = RouteValidator.from<P, Q, B>(routeSchemas);
     let requestParts: { params: P; query: Q; body: B };
-    if (isFakeRawRequest(req)) {
+    if (routeSchemas === undefined || isFakeRawRequest(req)) {
       requestParts = { query: {} as Q, params: {} as P, body: {} as B };
     } else {
-      const rawParts = CoreKibanaRequest.sanitizeRequest(req);
+      const routeValidator = RouteValidator.from<P, Q, B>(routeSchemas);
+      const rawParts = sanitizeRequest(req);
       requestParts = CoreKibanaRequest.validate(rawParts, routeValidator);
     }
```

Comment:
The validator will just return empty objects for each request part if the schemas are not defined. We can just short-circuit here as we already do for fake requests